Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -225,7 +275,7 @@ var TouchableHighlight = React.createClass({
 
   render: function() {
     return (
-      <View
+      <TVView
```

Comment:
If there's a way for the JS layer to know that it's running on tvOS, then I can make sure that TVView is only used in that case.  I see that there is the concept of "userInterfaceIdiom" in RCTPlatform.m, but that doesn't seem to be exposed in the JS layer.